README.md

```diff
@@ -60,9 +73,55 @@ tox
 
 ## Runtime Environment
 
-Install (from sources) system-wide:
+### Install pre-requirements
+Despite the pure python dependencies (see requirements.txt),
+Nmstate also needs NetworkManager to be running on the local system
+in order to configure the local network state.
+To access NetworkManager, Nmstate needs libnm and the corresponding
+introspection data (`NM-1.0.typelib`, provided by `python-gobject-base`).
+To manage OvS, Nmstate needs the packages `NetworkManager-ovs` and `openvswitch`.
+NetworkManager needs to be restarted in order to use the new configuration
+parameters (`conf.d/97-nmstate.conf`) and the OvS plugin.
+openvswitch service needs to be started.
+
+NetworkManager requires special configuration snippets to overcome some
+existing limitations.
+
+```shell
+yum -y install epel-release
+yum -y install \
+    dbus-python \
+    libibverbs \
+    NetworkManager \
+    NetworkManager-libnm \
+    NetworkManager-ovs \
+    openvswitch \
+    python-gobject-base \
+    python-ipaddress \
+    python-jsonschema \
+    python-setuptools \
+    python2-pyyaml \
+    python2-six
+echo -e "[device]\nmatch-device=*\nmanaged=0\n" >> \
+    /etc/NetworkManager/conf.d/97-nmstate.conf
+echo -e "[main]\nno-auto-default=*\n" >> \
+    /etc/NetworkManager/conf.d/97-nmstate.conf
+systemctl restart NetworkManager
```

Comment:
Is this the correct order?

Reply:
It does not matter, they are decoupled as fas as I know.
If this is not the case, it is a bug.